### PR TITLE
Widget Library Documentation Edit

### PR DESCRIPTION
**Pull request by jas-kas**
Adding new Issue Widget use cases.

### CODE REVIEW

File: src/docs/product/dashboards/widget-library/index.mdx [hunk 1]
```diff
@@ -69,26 +69,23 @@ The chart now shows cumulative counts at different response time thresholds.
...
 
-You can track the impact of your issues by counting the error events associated with each unique issue. Modify the "Issues For Review" widget by selecting the "All Events" data set. Update the search condition to filter for error events:
+You can track the impact of issues assigned to you or your team by counting the error events associated with each unique issue. Modify the "Issues For Review" widget by updating the search condition to filter for issue assignment:
 
-- Data Set: `All Events`
-- Search condition: `event.type:error`
+- Search condition: `is:unresolved is:for_review assigned_or_suggested:me`
 
-Update columns to group by `issues` and apply `count()` as the aggregating function. Order count in descending order so the widget displays issues with the most error events:
+Update columns to add `project` so you can see the associated project with each issue, and modify the “sort by” to `Events` so the widget displays issues with the most error events in descending order:
```
Comment (edwardgou-sentry) [inline src/docs/product/dashboards/widget-library/index.mdx:78]: I think the quote characters (") here have a different code than the standard quotes we use
```suggestion
Update columns to add `project` so you can see the associated project with each issue, and modify the "sort by" to `Events` so the widget displays issues with the most error events in descending order:
```

File: src/docs/product/dashboards/widget-library/index.mdx [hunk 2]
```diff
@@ -69,26 +69,23 @@ The chart now shows cumulative counts at different response time thresholds.
...
+The issues data set is unique in that you can add a free form [token](https://docs.sentry.io/product/sentry-basics/search/#syntax) to your search condition that will filter by issue titles containing the token. For example, you can search `integrity` to filter by all issue titles containing integrity. Update the search condition with a token of your choice:
 
-- Data Set: `All Events`
-- Search condition: `event.type:error`
+- Search condition: `is:unresolved is:for_review token`
 
-Update columns to group by `project` and apply `count()` and `count_unique(issue)` as the aggregating functions. Order count in descending order so the widget displays projects with the most error events:
+Update the “sort by” to Priority so the widget displays issues with that have been trending upward recently:
```
Comment (edwardgou-sentry) [inline src/docs/product/dashboards/widget-library/index.mdx:89]: Also different quotes here
```suggestion
Update the "sort by" to Priority so the widget displays issues with that have been trending upward recently:
```

File: src/docs/product/dashboards/widget-library/index.mdx [hunk 3]
```diff
@@ -69,26 +69,23 @@ The chart now shows cumulative counts at different response time thresholds.
...
-- Columns: `issue, count(), any(message)`
-- Sort by: `count() desc`
+- Columns: `issue, assignee, events, title, project`
+- Sort by: `Events`
 
-### Unique issues by projects
+### Priority issues
 
-Monitoring the number of unique issues alongside the total number of errors in your projects can give you a sense of the impact of issues in your project. For example, a low number of unique issues and a high number of total errors indicates you have a few high impact issues. Modify the "Issues For Review" widget by selecting the "All Events" data set. Update the search condition to filter to error events:
+The issues data set is unique in that you can add a free form [token](https://docs.sentry.io/product/sentry-basics/search/#syntax) to your search condition that will filter by issue titles containing the token. For example, you can search `integrity` to filter by all issue titles containing integrity. Update the search condition with a token of your choice:
```
Comment (edwardgou-sentry) [inline src/docs/product/dashboards/widget-library/index.mdx:85]: Should we also code block the last reference to `integrity` as well? I feel like it's a special term so it needs some sort of highlighting. Maybe quotes would be better if not code?
```suggestion
The issues data set is unique in that you can add a free form [token](https://docs.sentry.io/product/sentry-basics/search/#syntax) to your search condition that will filter by issue titles containing the token. For example, you can search `integrity` to filter by all issue titles containing `integrity`. Update the search condition with a token of your choice:
```

**Review by edwardgou-sentry — COMMENTED** [review]
Reviewing draft for @jas-kas. Content looks good to me! Just a few observations.

File: src/docs/product/dashboards/widget-library/index.mdx [hunk 4]
```diff
@@ -69,26 +69,23 @@ The chart now shows cumulative counts at different response time thresholds.
...
 
-### Unique issues by projects
+### Priority issues
 
-Monitoring the number of unique issues alongside the total number of errors in your projects can give you a sense of the impact of issues in your project. For example, a low number of unique issues and a high number of total errors indicates you have a few high impact issues. Modify the "Issues For Review" widget by selecting the "All Events" data set. Update the search condition to filter to error events:
+The issues data set is unique in that you can add a free form [token](https://docs.sentry.io/product/sentry-basics/search/#syntax) to your search condition that will filter by issue titles containing the token. For example, you can search `integrity` to filter by all issue titles containing `integrity`. Update the search condition with a token of your choice:
 
-- Data Set: `All Events`
-- Search condition: `event.type:error`
+- Search condition: `is:unresolved is:for_review token`
```
Comment (shruthilayaj) [inline src/docs/product/dashboards/widget-library/index.mdx:87]: Search condition: `is:unresolved is:for_review {token}`

**Review by shruthilayaj — APPROVED** [review]

**Review by imatwawana — APPROVED** [review]
Mostly style fixes, but otherwise looks good!

**Review by edwardgou-sentry — APPROVED** [review]

File: src/docs/product/dashboards/widget-library/index.mdx [hunk 5]
```diff
@@ -10,15 +10,15 @@ This feature is available only if your organization is on either a Business or T
 
 </Note>
 
-The widget library contains a collection of prebuilt widgets you can add to your [custom dashboards](/product/dashboards/customize-dashboards/). To access it, click on "Add Widget(s)" and switch to the "Widget Library" tab in the modal that appears.
+The widget library contains a collection of prebuilt widgets you can add to your [custom dashboards](/product/dashboards/customize-dashboards/). To access it, click on "Add Widget" and switch to the "Widget Library" tab in the modal that appears.
```
[the remark this answers is not in the document]
Comment (jas-kas) [reply src/docs/product/dashboards/widget-library/index.mdx:13]: The title of the modal is "Add Widget(s)" but I believe this is referring to clicking on the "Add Widget" button in the UI!

[the remark this answers is not in the document]
Comment (imatwawana) [reply src/docs/product/dashboards/widget-library/index.mdx:13]: Oh I totally didn't catch that. Unless I'm blind, we don't actually have a button that says "Add Widget" in the UI; there's just a plus icon. So I've updated my suggestion accordingly. But if I'm missing something in the UI, please let me know.

**Pull request merged**